### PR TITLE
Update SignUp UI

### PR DESCRIPTION
**Pull request by nehakumari2307**
Clean up SignUp page according to the wireframe

### CODE REVIEW

File: src/pages/SignUp.jsx [hunk 1]
```diff
@@ -17,13 +15,11 @@ export default function SignUp() {
...
 
   const { run: signUp, loading: signUpLoading } = useRequest(
-    // fixme: temp hard code company
+    // No harding for company required as it will be entered by user when registering for first time
 
     async (data) =>
       auth.signUp({
         ...data,
-        // fixme: temp hard code company
-        company: doc(database, "Companies", "KJLOQ9jWh9zsc3JfBugR"),
```
Comment (makersmelx) [inline src/pages/SignUp.jsx:26]: we still need this part for employer user. The company has to be a document reference to make jobs work. we will work on fixing the company next sprint. 
please help to delete the `company` field below

File: src/pages/SignUp.jsx [hunk 2]
```diff
@@ -17,13 +15,11 @@ export default function SignUp() {
...
       }),
     {
       manual: true,
```
Comment (makersmelx) [inline src/pages/SignUp.jsx:25]: for this, help add a success toast on success.
onError, instead of console, add a fail toast

**Review by makersmelx — CHANGES_REQUESTED** [review]

File: src/pages/SignUp.jsx (hunk 2 above)
Comment (nehakumari2307) [reply src/pages/SignUp.jsx]: Added toast message for success & failure

File: src/pages/SignUp.jsx (hunk 1 above)
Comment (nehakumari2307) [reply src/pages/SignUp.jsx]: Added the temp hardcode company piece back and disabled the company field on the signup page

**Review by makersmelx — APPROVED** [review]

**Pull request merged**